assets/scss/_tech-partner.scss

```diff
@@ -0,0 +1,238 @@
+    background: radial-gradient(236.22% 100.97% at 50% -4.33%, #0059D1 0%, #000000 100%);
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  }
+}
+
+.tech-partner__partnership-types {
+  font-size: 18px;
+  h2 {
+    font-size: 28px;
+    bottom: 20px;
+    position: relative;
+    @media only screen and (max-width: 768px) {
+      bottom: 30px;
+    }
+  }
+  h4,p {
+    font-size: 18px;
+  }
+  img {
+    margin-bottom: 10px;
+  }
+  .tech-partner__partnership-types-content {
+    .col-md-6 {
+      max-width: 939px;
+      height: 320px;
+      background: #FFFFFF;
+      box-shadow: 0px 4px 74px rgba(31, 49, 76, 0.25);
+      border-radius: 10px;
+      div {
+        width: 80%;
+        margin: 0 auto;
+        margin-top: 10%;
+      }
+    }
+  }
+  .tech-partner__partnership-types-content-two {
+    position: relative;
+    left: 30px;
+    @media only screen and (max-width: 768px) {
+      position: relative;
+      top: 30px;
+      left: 0;
+    }
+  }
+
+  .spacer125 {
+    @media only screen and (max-width: 768px) {
+      height: 200px;
+    }
+  }
+
+}
+
+.tech-partner__why, .tech-partner__more-benefits {
+  h2 {
+    font-size: 28px;
+  }
+  h4 {
+    font-size: 14px;
+  }
+
+  p {
+    font-size: 16px;
+  }
+  .col-xs-12 {
+    margin: 30px 0;
+    @media only screen and (max-width: 991px) {
+      margin: 20px auto;
+    }
+
+  }
+  h2, .col-xs-12 {
+    @media only screen and (max-width: 768px) {
+      margin: 20px auto;
+      width: 90%;
+    }
+    @media only screen and (max-width: 650px) {
+      width: 80%;
+    }
+  }
+  .spacer125 {
+    @media only screen and (max-width: 500px) {
+      height: 90px;
+    }
+  }
+}
+
+
+.tech-partner__who {
+
+  h2 {
+    font-size: 28px;
+  }
+
+  p {
+    font-size: 18px;
+  }
+
+  .col-md-6.list {
+    @media only screen and (max-width: 768px) {
+      margin-top: 30px;
+      text-align: left;
+      .spacer30 {
+        height: 25px;
+      }
+    }
+    p {
+      margin-left: 30px;
+      @media only screen and (max-width: 768px) {
+        margin-left: 0;
+        text-align: center;
+      }
+    }
+  }
+  .col-md-6 {
+    position: relative;
+    bottom: 10px;
+    @media only screen and (max-width: 768px) {
+      text-align: center;
+    }
+    ul {
+      list-style-type: none;
+      li {
+        @media only screen and (max-width: 768px) {
+          position: relative;
+          left: 12%;
+        }
+        @media only screen and (max-width: 500px) {
+          left: 0;
+        }
+        img {
+          float: left;
+          position: relative;
+          top: 15px;
+          right: 15px;
+          @media only screen and (max-width: 768px) {
+            right: 25px;
+          }
+        }
+        h4 {
+          font-size: 20px;
+          width: 110%;
+          @media only screen and (max-width: 500px) {
+            font-size: 14px;
+          }
+        }
+      }
+    }
+
+    .contact-us-link {
+      font-family: poppins;
```

Comment:
Maybe we can create a sass variable for the font-family

Reply:
added font variable